octoprint_mrbeam/static/js/app/view-models/material-store.js

```diff
@@ -36,12 +40,21 @@ $(function () {
             } else {
                 self.showConnectionError();
             }
+            self.mrBeamVM.removeNotifyIcon($("#materialstore_tab_btn"));
         };
 
         self.sendInitDetailsToMaterialStoreIframe = function () {
             self.sendMessageToMaterialStoreIframe(
                 MATERIAL_STORE_EVENT_TYPE.MR_BEAM_LOAD
             );
+
+            // send new laserhead model ID if changed
+            if (self.laserheadChangedVM.isLaserheadChanged()) {
+                self.sendMessageToMaterialStoreIframe("laserheadChanged", {
```

Comment:
Let's send the message here the same way we do it in the design store. for example: `event = 'selectFilter', payload = 'laserheadX'}`

Reply:
although check the material store frontend... I think we might have the event already defined there

Reply:
I checked but the current event is to only select product and not a filter; This is done.